subjects/devops/grades/README.md

```diff
@@ -0,0 +1,108 @@
+## grades
+
+### Instructions
+
+Create a script `grades.sh` which will ask the user for a specific number of students in order to evaluate them. After the number is set, the script will ask you to introduce the "names" and "grades" of the persons you wish to evaluate.
+
+The grades will have a range between 0 and 100 and will be numeric values only.
+
+According to each student grade you will have to return the following:
+
+- If the student grade is anything lower than 50 you will return the string "<name>: You had a poor performance!":
```

Comment:
For consistency maybe we can wrap the expected return in back tick.
"\<name>: You had a poor performance!" -> `"<name>: You had a poor performance!"`

to do for all messages

Reply:
done